src/main/python/image_preprocessing/template_matching.py

```diff
@@ -1,21 +1,27 @@
 import cv2
+from flask import jsonify
+import os
+DESKTOP = os.path.join(os.path.join(os.path.expanduser('~')), 'Desktop')
 
 
 class TemplateMatching:
-
     """
     The TemplateMatching class receives template images to identify the type of identification
     that is used in the image.
     Thus you provide it with templates and it will identify whether you used an id card, id book etc.
+    Line 16 that specifies my parameters are all pointing to the desktop my template folder needs
+    to be added to the site packages.
     """
 
-    def identify(src, template):
+    def identify(self, src, template=[(1034, DESKTOP + "/templates/temp_flag.jpg", 0.75, "idcard"),
```

Comment:
Remove the template argument. Rather add it as a private attribute of the `TemplateMatching` class in its constructor and use this private attribute further on. If you are unsure, ask @Nicvaniek how to do it correctly.